python/lsst/ts/mtdome/mock_llc/apscs.py

```diff
@@ -283,13 +569,23 @@ async def exit_fault(self, start_tai: float) -> float:
         `float`
             The expected duration of the command [s].
         """
-        durations = []
-        for i in range(NUM_SHUTTERS):
-            durations.append(self.shutter_motion[i].exit_fault(start_tai))
-        duration = max(durations)
-        self.end_tai = start_tai + duration
+        for shutter_id in range(NUM_SHUTTERS):
+            if True in self.drives_in_error_state[shutter_id]:
```

Comment:
You can use `any()`:

https://www.w3schools.com/python/ref_func_any.asp

This applies to other similar code.